tests/mpi_pi.f90

```diff
@@ -30,7 +30,7 @@ program calculatePi
 
   if (worldRank == 0) then
     call date_and_time(VALUES=timeValues)
-    write(*,"(A,I2,A,I2,A,I2.2)") "Start Time = ", timeValues(5), ":", timeValues(6), ":", timeValues(7)
+    write(*,"(A,I2,A,I2,A,I2.2)") "Start Time = ", timeValues(1), ":", timeValues(2), ":", timeValues(3), ":", timeValues(5), ":", timeValues(6), ":", timeValues(7)
```

Comment:
Currently this is what the output from the tests look like for the Start Time:
`Start Time = **: 1:04
:17:26:23`

The year shows up in `**` format, and the tests pass correctly.

Reply:
You are definitely on the right track.  You will need to update the string format in the `write` to accommodate the additional fields for the date stamp.  Right now, it is `(A,I2,A,I2,A,I2.2)` but you will need three additional integers and some other characters for the formatting.  The `**` is because the format for the year is `I2` instead of `I4`.  Also, the ` 1` instead of `01` is because that field is `I2` instead of `I2.2`.  I would shoot for having the date stamp in a format that will be easy to parse on the Python side.  Maybe something like: `dd/mm/yyy HH:MM:SS` or some other standardized string-based date format.